client/src/slices/user.slice.ts

```diff
@@ -0,0 +1,184 @@
+import _ from 'lodash';
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { ApiUser, ApiUserRole } from '../models/api-response';
+import { getLoggedInState } from '../utility/user-utils';
+import { AppState } from '../store';
+import { UserClient } from '../client/api';
+
+export interface UserState extends ApiUser {
+  activeRole?: ApiUserRole
+  isLoggedIn: boolean
+}
+
+export const userInitialState: UserState = {
+  edipi: '',
+  firstName: '',
+  lastName: '',
+  phone: '',
+  service: '',
+  email: '',
+  rootAdmin: false,
+  isRegistered: false,
+  userRoles: [],
+  isLoggedIn: false,
+};
+
+export interface UserRegisterData {
+  firstName: string
+  lastName: string
+  phone: string
+  service: string
+  email: string
+}
+
+export interface DashboardPayload {
+  orgId: number
+  workspaceId: number
+  dashboardUuid: string
+}
+
+const name = 'user';
+const addFavoriteDashboard = createAsyncThunk(`${name}/addFavoriteDashboard`,
```

Comment:
Can we drop the first arguments onto a new line, consistent with `orphaned-record.slice.ts`?